src/main/java/com/ironz/unsafe/UnsafeAndroid.java

```diff
@@ -467,6 +522,14 @@ public void putDouble(Object obj, long offset, double newValue) {
      * @since 1.8
      */
     @TargetApi(24)
+    public void copyMemory(Object srcObj, long srcAddr, Object dstObj, long dstAddr, long bytes) {
```

Comment:
Right but few method from original `sun.misc.Unsafe` which from ojluni (openjdk luni) has not been added by few Android architecture reasons. Do you have an tests which demonstrates successful methods invocation on Android O from this PR? Because as you can see original source code does not provide this api methods.